packages/protocol/contracts/identity/interfaces/IFederatedAttestations.sol

```diff
@@ -13,4 +35,19 @@ interface IFederatedAttestations {
       uint64[] memory,
       uint64[] memory
     );
+  function lookupIdentifiers(address, address[] calldata)
+    external
+    view
+    returns (uint256[] memory, bytes32[] memory);
+
+  function validateAttestationSig(
+    bytes32 identifier,
+    address issuer,
+    address account,
+    address signer,
+    uint64 issuedOn,
+    uint8 v,
+    bytes32 r,
+    bytes32 s
+  ) external view;
 }
```

Comment:
not sure why, but it seems like `public pure` functions are not included in interfaces

Reply:
also not sure, though there is one in `ICeloVersionedContract.sol`, maybe it makes sense for us to include the getAttestationHash function?

Reply:
sure, no harm in adding :)